mws/apis/easyship.py

```diff
@@ -32,7 +32,7 @@ class EasyShip(MWS):
     """Amazon EasyShip API.
 
     Docs:
-    http://docs.developer.amazonservices.com/en_IN/easy_ship/EasyShip_Overview.html
+    https://docs.developer.amazonservices.com/en_IN/easy_ship/EasyShip_Overview.html
```

Comment:
There are a number of small changes like this one, where simply `http` was changed to `https`. Adds a lot to the diff, but minor overall.